pennylane/qchem/factorization.py

```diff
@@ -317,8 +317,13 @@ def basis_rotation(one_electron, two_electron, tol_factor=1.0e-5):
         ops_l.append(ops_l_)
 
     ops = [ops_t] + ops_l
-    c_group = [op.coeffs for op in ops]
```

Comment:
Need to use `terms` explicitly to access `coeffs` and `ops` in the case of arithmetic operators.

Reply:
We did not have a test for `enable_new_opmath()` covering here?

Reply:
Nope, I don't think so.